brianlong/www.validators.app
# 186724765 new ping thing stats

#### What's this PR do?
- create user ping thing stats
- make charts smaller

#### How should this be manually tested?
- run migrations
- run tests
- start  `rails r daemons/process_ping_thing_daemon.rb`
- start  `rails r dev/keep_updating_pt_user_stats.rb` (this will be a schedule on prod/stage)
- visit ping things page
- add some ping things and check the user ping things table - there should be stats updating 

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186724765)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
